### ISSUE TITLE
is it possible to signal the end of a timer without sound? 

### ISSUE THREAD

**Issue by RedemptionC** [ivoronin/TomatoBar#74]
sometimes I just muted my Mac and I tend to miss the "notification" it's currently using, I want the signal to be more obvious

**Comment by SteveLauC** [comment]
Yeah, I would like to have a notification, I am interested in working on this is the author is ok with this feature!
